### PR TITLE
Feat - added slider component

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
* Added Slider component in my ui

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
looks good to merge

File: pages/slider.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            efficiently as possible.
+        </p>
+        <h3 class="heading heading-avatar">Simple Slider</h3>
+        <p class="description description-avatar">You can use <span class="colored-text"> slider </span> class for
+            headings as shown below. </p>
+        <iframe class="code-snippet-display"
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C59%2C48%2C1%29&t=base16-light&wt=none&l=htmlmixed&width=823&ds=false&dsyoff=82px&dsblur=100px&wc=true&wa=false&pv=27px&ph=22px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%252F%252F%2520Slider%250A%253Cinput%2520type%253D%2522range%2522%2520min%253D%25221%2522%2520max%253D%2522100%2522%2520value%253D%252250%2522%2520class%253D%2522slider%2522%2520id%253D%2522myRange%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <div class="badge_container-main flex-center">
```
Comment (LordUttam) [inline pages/slider.html:61]: change the name from badge container to something relevant.

**Review by LordUttam — COMMENTED** [review]
Rest looks good to merge

File: pages/slider.html (hunk 1 above)
Comment (Mohasin-Haque) [reply pages/slider.html:61]: This is the class used for particular styling on the section

**Pull request merged**